api/Dockerfile

```diff
@@ -37,9 +37,8 @@ RUN \
 ADD https://github.com/golang/go/raw/master/lib/time/zoneinfo.zip /zoneinfo.zip
 ENV ZONEINFO /zoneinfo.zip
 COPY --from=build /dist /
-ADD ./api/symphony-api.json /
-EXPOSE 8080
-EXPOSE 8081
-ENV LOG_LEVEL Debug
-# ENV CONFIG /symphony-api.json
-CMD sh -c 'if [ -f /etc/pki/ca-trust/source/anchors/proxy-cert.crt ]; then update-ca-trust; fi && exec /symphony-api -c $CONFIG -l $LOG_LEVEL'
+ADD ./api/symphony-api-no-k8s.json /
```

Comment:
If this is for your local test environment please don't check this in.

Reply:
This was a personal test environment setting.
I created a commit for deletion and the PR will be updated.